src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/DynamicViewTests.vb

```diff
@@ -2,6 +2,7 @@
 ' The .NET Foundation licenses this file to you under the MIT license.
 ' See the LICENSE file in the project root for more information.
 
+' These tests specifically cover late binding scenarios
```

Comment:
i'm surprised the *tests* need this though, as opposed to the test having strings in them that have VB code in them that use late binding.

Reply:
The tests use `ExpandoObject`